Clarify per-model eligibility for organization-level SAML SSO (BYOIdP) (#7085)
elastic/docs-content: deploy-manage/users-roles/cloud-organization/configure-saml-authentication.md

## Prerequisites [ec_prerequisites_4]

* You must have a SAML 2.0 compatible identity provider.

> * Eligibility for organization-level SAML SSO depends on your deployment model:

Reviewer: because these are evaluated per resource I think we need to be more specific. also ccm is not a deployment model

prefer "varies by" because people don't have a single resource type but instead a blend/collection

```suggestion
* Eligibility for organization-level SAML SSO varies by resource type:
```

> * * Eligibility for organization-level SAML SSO varies by resource type, rather than being a single, organization-wide entitlement:

Reviewer: ```suggestion
* Eligibility for organization-level SAML SSO varies by resource type, rather than being a single, organization-wide entitlement:
```

> * Eligibility for organization-level SAML SSO depends on the deployment model you use, rather than being a single, organization-wide entitlement:

Reviewer: ```suggestion
* * Eligibility for organization-level SAML SSO varies by resource type, rather than being a single, organization-wide entitlement:
```

    * **{{ech}}**: Requires an [Enterprise subscription](https://www.elastic.co/subscriptions/cloud).
    * **{{serverless-full}}**: Included at no additional cost, on any tier across all project types.

>     * **Self-managed clusters connected through [Cloud Connect](/deploy-manage/cloud-connect.md)**: Each connected cluster is evaluated individually against its own license. A cluster licensed Platinum or Enterprise is eligible, with no paid {{ecloud}} subscription required. Clusters on lower tiers, such as Basic, are not eligible, even if other eligible clusters are connected to the same organization.

Reviewer: "self-managed" has a very specific meaning in the docs that we need to account for (tl;dr very few things actually are "the same" between these deployment types) so we like to spell it out

is this plat/enterprise requirement for sso unique? EIS requires enterprise and autoops is free, so this is a different licensing model than other cloud connect features

as a side note, we don't really have cloud connect hinted at as a use case for SSO so we might want to add in a line or two about what you're SSOing TO in these cases - can follow with that later

```suggestion
    * **ECK, ECE, and self-managed clusters connected through [Cloud Connect](/deploy-manage/cloud-connect.md)**: Each connected cluster qualifies individually based on its license. A cluster with a Platinum or Enterprise license is eligible, with no paid {{ecloud}} subscription required. Clusters on lower tiers, such as Basic, are not eligible, even if other eligible clusters are connected to the same organization.
```
